modules/develop/pages/produce-data/leader-pinning.adoc

```diff
@@ -0,0 +1,51 @@
+= Leader Pinning
+:page-categories: Clients, Development
+// tag::single-source[]
+:description: Learn about leader pinning and how to configure a preferred partition leader location based on cloud availability zones or regions.
+
+Produce requests to write data to Redpanda topics go through the topic partition leader, which syncs messages across its follower replicas. For a Redpanda cluster deployed across multiple availability zones (AZs), leader pinning ensures that a topic's partition leaders are geographically closer to clients. Leader pinning helps decrease networking costs and guarantee lower latency.
+
+If consumers are located in the same preferred region or AZ for leader pinning, and you have not set up xref:develop:consume-data/follower-fetching.adoc[follower fetching], leader pinning can also help reduce networking costs on consume requests.
+
+== Prerequisites
+
+include::shared:partial$enterprise-license.adoc[]
+
+To enable leader pinning, you must xref:manage:rack-awareness.adoc#configure-rack-awareness[configure rack awareness] on the cluster. If the config_ref:enable_rack_awareness,true,properties/cluster-properties[] cluster configuration property is set to `false`, leader pinning is disabled across the cluster.
+
+== Configure leader pinning
+
+You can use a topic configuration property as well as a cluster configuration property to configure leader pinning.
+
+The following steps assume that you have Redpanda deployed in a multi-AZ or multi-region cluster, and you have configured each broker so that the config_ref:rack,true,properties/broker-properties[] configuration property contains rack IDs corresponding to the AZ IDs.
```

Comment:
```suggestion
The following steps are based on the assumption that you have Redpanda deployed in a multi-AZ or multi-region cluster, and you have configured each broker so that the config_ref:rack,true,properties/broker-properties[] configuration property contains rack IDs corresponding to the AZ IDs.
```